room/runtime/src/main/java/androidx/room/DatabaseConfiguration.java

```diff
@@ -291,6 +295,7 @@ public DatabaseConfiguration(@NonNull Context context, @Nullable String name,
             @NonNull SupportSQLiteOpenHelper.Factory sqliteOpenHelperFactory,
             @NonNull RoomDatabase.MigrationContainer migrationContainer,
             @Nullable List<RoomDatabase.Callback> callbacks,
+            @Nullable List<RoomDatabase.PrepackagedCallback> prepackagedCallbacks,
```

Comment:
To be backwards compatible with old generated code, even though this API is marked with `@RestrictTo` we need to copy this constructor and add the new parameter and then update all other constructors to call the newest one with `null`. Here is an example: https://github.com/androidx/androidx/commit/1263185ef8c50d4059758e382610345a24f82c53#diff-a135d07f04ca20de890a27db682fe8e2